yt_dlp/extractor/afreecatv.py

```diff
@@ -61,46 +61,46 @@ def _perform_login(self, username, password):
 
     def _call_api(self, endpoint, display_id, data=None, headers=None, query=None):
         return self._download_json(Request(
-            f'https://api.m.afreecatv.com/{endpoint}',
+            f'https://api.m.sooplive.co.kr/{endpoint}',
             data=data, headers=headers, query=query,
             extensions={'legacy_ssl': True}), display_id,
             'Downloading API JSON', 'Unable to download API JSON')
 
 
 class AfreecaTVIE(AfreecaTVBaseIE):
-    IE_NAME = 'afreecatv'
-    IE_DESC = 'afreecatv.com'
+    IE_NAME = 'soop'
+    IE_DESC = 'sooplive.co.kr'
     _VALID_URL = r'''(?x)
                     https?://
                         (?:
-                            (?:(?:live|afbbs|www)\.)?afreeca(?:tv)?\.com(?::\d+)?
+                            (?:(?:live|afbbs|www)\.)?sooplive\.co\.kr(?::\d+)?
                             (?:
                                 /app/(?:index|read_ucc_bbs)\.cgi|
                                 /player/[Pp]layer\.(?:swf|html)
                             )\?.*?\bnTitleNo=|
```

Comment:
do links like these even exist anymore? it seems like all VODs are on the `vod` subdomain now, no?

if we want to keep this part of the regex pattern for legacy reasons or something, we should turn back the clock on the domain name here too
```suggestion
                            (?:(?:live|afbbs|www)\.)?afreecatv.\com(?::\d+)?
                            (?:
                                /app/(?:index|read_ucc_bbs)\.cgi|
                                /player/[Pp]layer\.(?:swf|html)
                            )\?.*?\bnTitleNo=|
```